docs/environment-variables.md

```diff
@@ -43,13 +43,13 @@ Email verification is when a user registers into the application and to be able
 
 | Environment Variable | Description                                                                                                     | Example                |
 |----------------------|-----------------------------------------------------------------------------------------------------------------|------------------------|
-| MAIL_DEFAULT_SENDER  | Email used to send email verification emails. This is in string format.                                         | 'some_username@gmail.com' |
+| MAIL_DEFAULT_SENDER  |Email used to send email verification emails. This has to be a real email address that you've got access to. For example , johnclark@gmail.com . This is in string format.                                       | 'some_username@gmail.com' |
```

Comment:
@vj-codes there is already an example tab here, then should we include the example in description too?

Reply:
@devkapilbansal the issue suggested adding the example in the description as well just to make things clearer , what say?

Reply:
@devkapilbansal can you update the labled do you still need to remove the example from description 👀